test/system/application-scans/node.spec.ts

```diff
@@ -216,21 +216,19 @@ describe("node application scans", () => {
     expect(depGraphNpmFromGlobalNodeModules.rootPkg.name).toEqual("lib");
   });
 
-  it("should extract image content successfully", async () => {
+  it("should collect application files successfully", async () => {
     const fixturePath = getFixture("npm/multi-project-image.tar");
     const imageNameAndTag = `docker-archive:${fixturePath}`;
-    const result = await extractContent([getNodeAppFileContentAction], {
+    const result = await collectApplicationFiles({
       path: imageNameAndTag,
     });
-    expect(Object.keys(result.extractedLayers).length).toEqual(608);
-    Object.keys(result.extractedLayers).forEach((fileName) => {
-      expect(
-        fileName.endsWith("/package.json") ||
-          fileName.endsWith("/package-lock.json") ||
-          fileName.endsWith("/yarn.lock"),
-      ).toBeTruthy();
-      expect("node-app-files" in result.extractedLayers[fileName]).toBeTruthy();
-    });
+    const applicationFilesFact: ApplicationFilesFact =
+      result.scanResults[0].facts.find(
+        (fact) => fact.type === "applicationFiles",
+      )!.data;
+
+    const fileList = applicationFilesFact[0].fileHierarchy.map((fh) => fh.path);
+    expect(fileList.length).toEqual(4);
```

Comment:
Please validate on the specific file names

Reply:
Done ✌️